spec/models/routed_query_keyword_spec.rb

```diff
@@ -34,6 +34,18 @@
     end
   end
 
+  describe 'creating a keyword which contains html' do
```

Comment:
Two more minor tweaks: 
- move this spec within the `Creating new instance` block
- use `context` rather than `describe`:
```ruby
context 'when the keyword contains html' do
```

Reply:
Done.